<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatchSettingsPanel</class>
 <widget class="QDialog" name="PatchSettingsPanel">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patches configuration</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Skipping</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox_skiphudmemos">
          <property name="toolTip">
           <string>This will make the game not pause when you collect an upgrade or expansion</string>
          </property>
          <property name="text">
           <string extracomment="To skip showing a message that displays &quot;you've collected an item!&quot;">Skip item collected messages</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_skipfrigate">
          <property name="text">
           <string>Skip Pirate Frigate</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_skipcrater">
          <property name="text">
           <string>Skip Impact Crater</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Difficulty</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <widget class="QRadioButton" name="radioStackProtection">
          <property name="toolTip">
           <string>Combine the obtained suits' defenses</string>
          </property>
          <property name="text">
           <string extracomment="The individual protection given by all the suits is combined">Stack with suits</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string extracomment="Protection against extreme heat">Heat damage protection:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string extracomment="Stronger suits will have better defenses and thus cause a reduction in taken damage">Damage reduction:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="radioStaggerProtection">
          <property name="toolTip">
           <string>Use the total damage reduction when you normally get the suits in sequence</string>
          </property>
          <property name="text">
           <string extracomment="Always use the most resilient/powerful suit">Always use most powerful</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="radioHeatDamageAny">
          <property name="text">
           <string>Any except Power Suit</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="radioHeatDamageVaria">
          <property name="text">
           <string>Varia Suit only</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox_bomblockpick">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string extracomment="This means to enable using the Power bomb as a tool to force a door to open regardless of its color">Power bomb lock-picking</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkBox_supermetroiddoors">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Super Metroid-like doors</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Door modifications (experimental)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox_enablevaultdoor">
        <property name="toolTip">
         <string>This will enable randomizing the door leading to the Vault in Chozo Ruins' Main Plaza.</string>
        </property>
        <property name="text">
         <string>Enable one-way door randomization</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_enableverticaldoor">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Enable vertical door randomization</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Misc</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_fixflaaghra">
        <property name="text">
         <string extracomment="Either &quot;to fix broken music files&quot; or &quot;to use the full version of Flaahgra's song&quot;">Fix Flaahgra music files</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string extracomment="Where is the Metroid Prime Trilogy disk?">Metroid Prime Trilogy disk:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineFlaaghra">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Either American or European version</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="browseButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_keepattractmode">
        <property name="text">
         <string extracomment="To keep a series of videos that can be seen in the title screen after not pressing anything for a while.">Keep Attract mode</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PatchSettingsPanel</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PatchSettingsPanel</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
